scripts/install_foxx.sh

```diff
@@ -138,7 +138,7 @@ fi
 basic_auth="$local_DATABASE_USER:$local_DATAFED_DATABASE_PASSWORD"
 url="http://${local_DATAFED_DATABASE_HOST}:${local_DATABASE_PORT}/_api/database/user"
 # Do not output to /dev/null we need the output
-code=$(curl -s -o /dev/null -w "%{http_code}" --user "$basic_auth" "$url")
+code=$(LD_LIBRARY_PATH="${DATAFED_DEPENDENCIES_INSTALL_PATH}:$LD_LIBRARY_PATH" curl -s -o /dev/null -w "%{http_code}" --user "$basic_auth" "$url")
```

Comment:
This would be a good suggestion, but doing this with PATH and LD_LIBRARY_PATH in docker containers appears to introduce instability into the build process when scripts are sourced and interfere with the build. https://github.com/apptainer/singularity/issues/3510